src/Sylius/Component/Media/Model/ImageInterface.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Media\Model;
+
+use Symfony\Cmf\Bundle\MediaBundle\ImageInterface as CmfImageInterface;
+
+/**
+ * Interface for the model representing an Image entity which is referencing an Image cmf document.
+ *
+ * @author Aram Alipoor <aram.alipoor@gmail.com>
+ */
+interface ImageInterface
+{
+    /**
+     * @return string
+     */
+    public function getMediaId();
+
+    /**
+     * @param string $mediaId
+     */
+    public function setMediaId($mediaId);
```

Comment:
You can read my [updated comment](https://github.com/Sylius/Sylius/pull/2975#issue-93486115) ;)